README.md

```diff
@@ -253,6 +253,9 @@ Your Space ID.
 #### host (default: `'cdn.contentful.com'`)
 Set the host used to build the request URI's.
 
+#### basePath (default: ``)
```

Comment:
Because this also forces to upgrade to the latest sdk-core version which includes that feature